.github/workflows/main.yml

```diff
@@ -388,6 +395,52 @@ jobs:
       run:
         python3 `which scons` test show_long_tests=yes verbose_tests=yes --debug=time
 
+  windows-2022:
+    name: ${{ matrix.os }}, MSVC ${{ matrix.vs-toolset }}, Python ${{ matrix.python-version }}
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 60
+    defaults:
+      run:
+        shell: bash -l {0}
+    strategy:
+      matrix:
+        os: ["windows-2022"]
+        vs-toolset: ["14.3"] # 'cl' is not recognized for earlier toolsets
+        python-version: [ "3.7", "3.8", "3.9", "3.10" ]
+      fail-fast: false
+    steps:
+    - uses: actions/checkout@v2
+      name: Checkout the repository
+      with:
+        submodules: recursive
+    - name: Set up conda
+      uses: conda-incubator/setup-miniconda@v2
+      with:
+        auto-update-conda: true
+        python-version: ${{ matrix.python-version }}
+        mamba-version: "*"
+        activate-environment: test
+        channels: conda-forge,defaults
+        channel-priority: true
+    - name: Install conda dependencies
+      # See https://github.com/conda-forge/boost-cpp-feedstock/issues/41 for why we
+      # use boost-cpp rather than boost from conda-forge
+      run: |
+        mamba install -q scons numpy cython ruamel_yaml boost-cpp eigen yaml-cpp h5py pandas pytest
+      shell: pwsh
+    - name: Build Cantera
+      run: scons build CXX=cl CC=cl system_eigen=y system_yamlcpp=y VERBOSE=True
```

Comment:
You shouldn't have to specify `CXX` and `CC` here, SCons should do the "right thing" automatically. If these have to be specified then it's a smell to me that something else is wrong.

Reply:
True. Tried that to force using something that wasn't found for the toolsets that aren't installed.